onnxruntime/core/providers/mkldnn/mkldnn_provider_factory.cc

```diff
@@ -1,38 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
how come this file shows as deleted (instead of renamed)
hopefully we can maintain the history

Reply:
I actually renamed the file and sub-folder. But git committed this change as deleted and new file.